fi.laamaa.m8c.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: fi.laamaa.m8c
```

Comment:
I see no mention anywhere of `laamaa.fi`. Is this domain controlled by the developers?

Reply:
This is the developers website address according to their GitHub profile. I assumed this was the right thing to use here but please let me know if it's not.

Reply:
it's not about the developers (as a person) but rather the project. Something based on github repository would be more appropriate here

See https://docs.flatpak.org/en/latest/conventions.html#application-ids